lock-api/src/main/java/com/palantir/lock/client/RemoteTimelockServiceAdapter.java

```diff
@@ -62,7 +63,8 @@ public static RemoteTimelockServiceAdapter create(
                 rpcClient,
                 conjureClient,
                 new LegacyLeaderTimeGetter(conjureClient),
-                RequestBatchersFactory.create(lockWatchEventCache, namespace, Optional.empty()));
+                RequestBatchersFactory.create(
+                        lockWatchEventCache, ValueCacheUpdater.NO_OP, namespace, Optional.empty()));
```

Comment:
Checked that everywhere that calls this has a no-op event cache. In that case we may as well surely just inline the no op event cache here too?